build/fbcode_builder/CMake/FindGflags.cmake

```diff
@@ -45,6 +45,16 @@ if (gflags_FOUND)
   # cmake module compat
   set(GFLAGS_FOUND ${gflags_FOUND})
   set(Gflags_FOUND ${gflags_FOUND})
+  set(FOLLY_HAVE_LIBGFLAGS ${gflags_FOUND})
+  if (TARGET gflags-shared)
+    list(APPEND FOLLY_SHINY_DEPENDENCIES gflags-shared)
+  elseif (TARGET gflags)
+    list(APPEND FOLLY_SHINY_DEPENDENCIES gflags)
+  else()
+    message(FATAL_ERROR "Unable to determine the target name for the GFlags package.")
+  endif()
+  list(APPEND CMAKE_REQUIRED_LIBRARIES ${GFLAGS_LIBRARIES})
+  list(APPEND CMAKE_REQUIRED_INCLUDES ${GFLAGS_INCLUDE_DIR})
```

Comment:
Are these really right?  Current gflags cmake config packages look like they export gflags_LIBRARIES and gflags_INCLUDE_DIR, not GFLAGS_LIBRARIES and GFLAGS_INCLUDE_DIRECTORIES.